src/command_modules/azure-cli-resource/azure/cli/command_modules/resource/commands.py

```diff
@@ -222,7 +222,7 @@ def load_command_table(self, _):
     with self.command_group('deployment', resource_deployment_sdk) as g:
         g.command('list', 'list_at_subscription_scope', table_transformer=transform_deployments_list, min_api='2018-05-01')
         g.command('show', 'get_at_subscription_scope', exception_handler=empty_on_404)
-        g.command('delete', 'delete_at_subscription_scope')
+        g.command('delete', 'delete_at_subscription_scope', supports_no_wait=True)
```

Comment:
do we need a `wait` command here?

Reply:
Ah, thanks for the catch. Yes, I'll add it.